environment/learnrails/calculator.rb

```diff
@@ -0,0 +1,26 @@
+class Calculator
```

Comment:
please add rspec test for class Calculator
e.g. https://dev.to/mmcclure11/basic-ruby-calculator-with-rspec-5fjn